app/views/govuk_publishing_components/components/_breadcrumbs.html.erb

```diff
@@ -16,22 +16,26 @@
   <%= raw JSON.pretty_generate(breadcrumb_presenter.structured_data) %>
 </script>
 
-<div class="<%= classes.join(" ") %>" data-module="gem-track-click">
+<nav class="<%= classes.join(" ") %>" data-module="gem-track-click" aria-label="Breadcrumb">
   <ol class="govuk-breadcrumbs__list">
     <% breadcrumbs.each_with_index do |crumb, index| %>
       <% breadcrumb = GovukPublishingComponents::Presenters::Breadcrumb.new(crumb, index) %>
         <li class="govuk-breadcrumbs__list-item">
         <% if breadcrumb.is_link? %>
+          <% if index == breadcrumbs.size - 1 %>
+            <% last_link = index ? "page" : "" %>
+          <% end %>
           <%= link_to(
             breadcrumb[:title],
             breadcrumb.path,
             data: breadcrumb.tracking_data(breadcrumbs.length),
             class: "govuk-breadcrumbs__link",
+            "aria-current": last_link,
```

Comment:
Just a note here in case this PR does wind up getting carried forward: this is not quite right for our implementation of breadcrumbs. We don't show the current page in the list, as per the design system. The design system can add a current page (though it's not covered as an example in their docs) but it's not implemented here, we only show parent items. When the current page does exist in Design System breadcrumbs it's a list item but not a link.